.npmignore

```diff
@@ -1 +1,4 @@
-src
```

Comment:
Most of people now compile their scripts (me included) so it is good to let them use ES6 directly